game.cpp

```diff
@@ -1351,18 +1326,72 @@ void test_game() //!OCLINT tests may be many
     assert(player_copy.get_speed() == p.get_speed());
   }
 
-  /// When a player is killed it stays in the player vector but its state is dead
+  /// When a player is out it stays in the player vector but its state is out
   {
     game g;
 
     auto num_of_players_begin = g.get_v_player().size();
 
-    //kill the first player
     g.kill_player(0);
 
     assert(num_of_players_begin == g.get_v_player().size());
-    assert(is_dead(g.get_player(0)));
+    assert(is_out(g.get_player(0)));
+  }
+
+#ifdef FIX_ISSUE_606
+  {
+    // (626) A player that is out can be revived
+    game g;
+    g.kill_player(0);
+    assert(is_out(g.get_player(0)));
+    g.revive_player(0);
+    assert(!is_out(g.get_player(0)));
+  }
+#endif
+
+#ifdef FIX_ISSUE_611
+  {
+    // (611) A player that is out revives after some time
+    game g;
+    g.kill_player(0);
+    const int revive_time = 100;
```

Comment:
Where does `revive_time` come from? Either add `// just a high number` or use a function such as `get_default_revive_time`.

Reply:
Agreed; I think I mention in the issue that the nb is arbitrary, but not in code yet